themes/digital.gov/layouts/partials/core/attribution.html

```diff
@@ -0,0 +1,17 @@
+{{/* Gets the id of the header image and cleans it up */}}
+{{- $thisuid := cond .IsNode .Params.Image .Parent.Params.Image -}}
+{{- $thisuid := replaceRE "\\.[jpegpn]{3,4}$" "" $thisuid -}}
+
+{{/* Matches the id of the header image to a .yml file */}}
+{{- $thisimg := index $.Site.Data.images $thisuid -}}
+
+
+<p class="dg-attribution">
+  {{- if $thisimg.credit -}}
+    Credit:&nbsp
+    {{ $thisimg.credit | markdownify }}
+  {{- else if $thisimg.caption -}}
+    Credit:&nbsp
+    {{ $thisimg.caption | markdownify }}
+  {{- end -}}
```

Comment:
Unsure about the caption being used as a credit, but we could simplify to add the label once.

```suggestion
  Credit:&nbsp

  {{- if $thisimg.credit -}}
    {{ $thisimg.credit | markdownify }}
  {{- else if $thisimg.caption -}}
    {{ $thisimg.caption | markdownify }}
  {{- end -}}
```

Reply:
Yeah this is necessary because a lot of the images but the credit in the caption field 🤦 
(see https://digital.gov/images/ for examples)

Reply:
That's unfortunate. Do you know if we have a card in the backlog for this issue?

Reply:
Created a card for [this](https://trello.com/c/PXNQZ50b/2395-dg-update-images-to-put-attribution-in-credit-field-rather-than-caption). Could possibly be a workflow issue seeing as most of the images are labeled this way...